perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -104,6 +104,10 @@
                      'Only applies to the api_multistream scenario.',
                      lower_bound=1)
 
+flags.DEFINE_integer('object_storage_list_consistency_iterations', 200,
```

Comment:
consider add the following help text to the flag:

"this flag is introduced mainly for regression test purpose. Reduce this number to shorten execution time of the api_namespace scenario. However, to get useful metrics from the api_namespace scenario, a high number of iterations should be used (>=200)"